common/subtitle-reader/subtitle-reader.ts

```diff
@@ -58,18 +58,21 @@ const sortVttCues = (list: VTTCue[]) => {
 export default class SubtitleReader {
```

Comment:
Can you move the ruby parsing into this class instead of handling it in `SubtitlePlayer.tsx` and `subtitle-controller.ts`? I'm worried that if we put that type of parsing code too far outside this class then it'll have undesireable interactions with other features that do stuff to the subtitle text, like the [coloring feature ](https://github.com/killergerbah/asbplayer/pull/813) Shanary is working on.

It looks like you sort of started on that approach already in this file.